### PR TITLE
Turn on amp-consent-geo-override in canary and prod

### PR DESCRIPTION
**Pull request by micajuine-ho**

### CODE REVIEW

File: extensions/amp-consent/0.1/test/test-amp-consent.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ describes.realWin(
       ampdoc = env.ampdoc;
       win = env.win;
       toggleExperiment(win, 'amp-consent-v2', true);
+      toggleExperiment(win, 'amp-consent-geo-override', true);
```
Comment (lannka) [inline extensions/amp-consent/0.1/test/test-amp-consent.js]: let's keep those experiments toggling. they should be removed together with the flags once fully launched.

File: extensions/amp-consent/0.1/test/test-amp-consent.js [hunk 2]
```diff
@@ -755,7 +739,7 @@ describes.realWin(
         defaultConfig = dict({
           'consents': {
             'ABC': {
-              'checkConsentHref': 'https://response1',
+              'checkConsentHref': 'https://geo-override-check2/',
```
Comment (lannka) [inline extensions/amp-consent/0.1/test/test-amp-consent.js]: does this matter? it is a unit test, are we making request to local server?

Comment (micajuine-ho) [reply extensions/amp-consent/0.1/test/test-amp-consent.js]: It doesn't matter if we keep experiments toggling

File: extensions/amp-consent/0.1/test/test-amp-consent.js (hunk 1 above)
Comment (micajuine-ho) [reply extensions/amp-consent/0.1/test/test-amp-consent.js]: ok

File: build-system/global-configs/prod-config.json [hunk 3]
```diff
@@ -1,5 +1,8 @@
 {
-  "allow-doc-opt-in": ["amp-next-page"],
```
Comment (lannka) [inline build-system/global-configs/prod-config.json]: revert

File: extensions/amp-consent/0.1/amp-consent.js [hunk 4]
```diff
@@ -467,7 +467,10 @@ export class AmpConsent extends AMP.BaseElement {
           return Promise.resolve(consentRequired);
         }
         return this.getConsentRemote_().then(consentResponse => {
-          return !!consentResponse['consentRequired'];
+          return (
+            !!consentResponse['consentRequired'] ||
```
Comment (lannka) [inline extensions/amp-consent/0.1/amp-consent.js]: we should only use `promptIfUnknown` when `consentRequired = undefined`

Comment (lannka) [reply extensions/amp-consent/0.1/amp-consent.js]:
```js
let consentRequired = consentResponse['consentRequired'];
if (consentRequired === undefined) {
  consentRequired = !!consentResponse['promptIfUnknown'];
}
return consentRequired;
```

Comment (micajuine-ho) [reply extensions/amp-consent/0.1/amp-consent.js]: done

File: extensions/amp-consent/0.1/amp-consent.js [hunk 5]
```diff
@@ -467,7 +467,10 @@ export class AmpConsent extends AMP.BaseElement {
           return Promise.resolve(consentRequired);
         }
         return this.getConsentRemote_().then(consentResponse => {
-          return !!consentResponse['consentRequired'];
+          // `promptIfUnknown` is a legacy field
+          return consentResponse['consentRequired'] !== undefined
+            ? !!consentResponse['consentRequired']
+            : !!consentResponse['promptIfUnknown'];
```
Comment (zhouyx) [inline extensions/amp-consent/0.1/amp-consent.js]: Please add a comment and also document in readme later, that when a response cannot be found, amp-consent will ignore the prompt and unblock components. Publishers need to be extra careful with this.

Comment (zhouyx) [reply extensions/amp-consent/0.1/amp-consent.js]: And we can go ahead and remove legacy method in the next PR : )

**Pull request merged**